package/MDAnalysis/analysis/distances.py

```diff
@@ -192,6 +193,6 @@ def between(group, A, B, distance):
     from MDAnalysis.core.AtomGroup import AtomGroup
 
     ns_group = AtomNeighborSearch(group)
-    resA = set(ns_group.search_list(A, distance))
-    resB = set(ns_group.search_list(B, distance))
+    resA = set(ns_group.search(A, distance))
```

Comment:
My suspicion for the original issue was that I rebased against develop, but it wasn't actually the tip (most recent commit) of develop, and apparently that will mess up the diff that shows up here. And indeed the github diffs here are now showing all kinds of stuff that was merged recently, so it definitely looks like a rebase against an old commit level of develop.